tests/lib/tools/fs-state

```diff
@@ -1,25 +1,46 @@
 #!/bin/bash
 
 show_help() {
-    echo "usage: fs-state mock-dir [dirname]"
+    echo "usage: fs-state mock-dir <--new> [dirname]"
```

Comment:
Do we need --new in this PR or could it be a separate one?

Reply:
well, yes because there is a test wich fails in case we copy the file /dev/input/event1

Reply:
this code has changed